index.html

```diff
@@ -52,7 +53,170 @@ <h1>I'm CAkE <br>
+                <div class="button-list">
+                    <ul>
+                        <li><a href="">HTML</a></li>
+                        <li><a href="">CSS</a></li>
+                        <li><a href="">JAVASCRIPT</a></li>
+                    </ul>
+                </div>
+                <div class="btn">
+                    <button type="button">See project</button>
+                </div>
+            </div>
+
+
+            <div class="Projects">
+                <img class="work-banners" src="Images&Icons/work3.png" alt="">
+                <h2 class="Tonic">Tonic</h2>
+                <div class="desc-list">
+                    <ul>
+                        <li class="CANOPY">CANAPY</li>
+                        <li><a href=""><img src="Images&Icons/Counter.png" alt=""></a></li>
+                        <li>Back End Dev</li>
+                        <li><a href=""><img src="Images&Icons/Counter.png" alt=""></a></li>
+                        <li>2022</li>
+                    </ul>
+                </div>
+
+                <p>A daily selection of privately personalized reads; no accounts
+                    sign-ups required
+                </p>
+                <div class="button-list">
+                    <ul>
+                        <li><a href="">HTML</a></li>
+                        <li><a href="">CSS</a></li>
+                        <li><a href="">JAVASCRIPT</a></li>
+                    </ul>
+                </div>
+                <div class="btn">
+                    <button type="button">See project</button>
+                </div>
+
+                <div class="Projects">
+                    <img class="work-banners" src="Images&Icons/work4.png" alt="">
+                    <h2 class="Tonic">Multi-post Stories</h2>
+                    <div class="desc-list">
+                        <ul>
+                            <li class="CANOPY">CANAPY</li>
+                            <li><a href=""><img src="Images&Icons/Counter.png" alt=""></a></li>
+                            <li>Back End Dev</li>
+                            <li><a href=""><img src="Images&Icons/Counter.png" alt=""></a></li>
+                            <li>2022</li>
+                        </ul>
+                    </div>
+
+                    <p>A daily selection of privately personalized reads; no accounts
+                        sign-ups required
+                    </p>
+                    <div class="button-list">
+                        <ul>
+                            <li><a href="">HTML</a></li>
+                            <li><a href="">CSS</a></li>
+                            <li><a href="">JAVASCRIPT</a></li>
+                        </ul>
+                    </div>
+                    <div class="btn">
+                        <button type="button">See project</button>
+                    </div>
+                </div>
+
 
+
+
+
+
+
+            </div>
+
+        </section>
+        <section class="About">

```

Comment:
- [ ] Kindly work on the styling check Figma template
- Your Design
![Capturesty](https://user-images.githubusercontent.com/81444338/176996175-69412e84-79b5-45ec-9346-c974cbe3aeef.PNG)
- Figma Design
![Capturestyl](https://user-images.githubusercontent.com/81444338/176996198-0daeba67-066e-4100-a791-222b60acb187.PNG)